docs/usage/user-stories/swissquote.md

```diff
@@ -197,18 +199,42 @@ Here is the dashboard for our current scheduler:
 
 <figure markdown>
   ![Swissquote scheduler dashboard](../assets/images/swissquote_stats.png){ loading=lazy }
-  <figcaption>A dashboard we made at Swissquote to keep our Renovate runs in check, July 2022.</figcaption>
+  <figcaption>A dashboard we made at Swissquote to keep our Renovate runs in check, November 2023.</figcaption>
 </figure>
 
 We don’t force any team to use Renovate, each team can decide to opt-in and do it for each project separately.
 
 Some statistics:
 
-- 824 repositories enabled out of about 2000 active repositories
-- 8000 PRs were merged since we installed Renovate
+- 857 repositories enabled out of about 2000 active repositories
+- 11000 PRs were merged since we installed Renovate
 - 239 PRs were merged last month
 - 2 SSDs died on our Renovate machine with the number of projects to clone again and again
 
+### How does the scheduler work ?
+
+The scheduler is a Node.js application which handles an in-memory queue and starts Docker containers to run Renovate on.
+Our custom scheduler application regularly sends points to an InfluxDB which we then display in Grafana.
```

Comment:
```suggestion
Our custom scheduler application regularly sends data points to our InfluxDB database, which we then display in Grafana.
```

- Maybe data points is clearer than just points?
- Might be good to mention that InfluxDB is a database?
- Add a comma to separate the parts about sending data, and displaying the data

Reply:
Yes indeed it's better to put data points